cilium-health/launch/endpoint.go

```diff
@@ -192,6 +192,10 @@ func LaunchAsEndpoint(owner endpoint.Owner, hostAddressing *models.NodeAddressin
 
 	ip4 := node.GetIPv4HealthIP()
 	ip6 := node.GetIPv6HealthIP()
+	ip4MaxMask := net.CIDRMask(32, 32)
+	ip6MaxMask := net.CIDRMask(128, 128)
```

Comment:
Can you put those variables in `pkg/defaults/node`? This is the file where we usually store default masks. I would give those variables names like `HealthIPv4Mask` and `HealthIPv6Mask`.

Reply:
@mrostecki I was looking for that, there is defaults.ContainerIPv{4|6}Mask